itou/siae_evaluations/models.py

```diff
@@ -665,10 +666,44 @@ def state_from(criteria):
 
         return max(
             (state_from(criteria) for criteria in self.evaluated_administrative_criteria.all()),
-            key=STATES_PRIORITY.index,
+            key=self.STATES_PRIORITY.index,
             default=evaluation_enums.EvaluatedJobApplicationsState.PENDING,
         )
 
+    def hide_state_from_siae(self):
+        """Hide in-progress evaluation from SIAE, until results are official."""
+        if self.evaluated_siae.submission_freezed_at is None:
+            return False
+        adversarial_stage_start = self.evaluated_siae.evaluation_campaign.calendar.adversarial_stage_start
+        phase2bis = (
+            timezone.localdate()
+            <=
+            # submission_freezed_at is reset with EvaluationCampaign.transition_to_adversarial_phase, which immediately
+            # shows their state to SIAEs.
+            # On the day of the transition, keep phase 2bis active so that SIAE don’t see their state until the admin
+            # action in the admin to transition to adversarial stage is performed.
+            adversarial_stage_start
+        )
+        if phase2bis:
+            return True
```

Comment:
```suggestion
        # Check if still in phase 2bis
        if (
            timezone.localdate()
            <=
            # submission_freezed_at is reset with EvaluationCampaign.transition_to_adversarial_phase, which immediately
            # shows their state to SIAEs.
            # On the day of the transition, keep phase 2bis active so that SIAE don’t see their state until the admin
            # action in the admin to transition to adversarial stage is performed.
            self.evaluated_siae.evaluation_campaign.calendar.adversarial_stage_start
        ):
            return True
```